packages/components/datepicker/src/Calendar/CalendarCaptionLabel.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { CaptionLabelProps, useDayPicker } from 'react-day-picker';
+import { Heading } from '@contentful/f36-typography';
+import { getStyles } from './Calendar.styles';
+import { cx } from 'emotion';
+
+export const CalendarCaptionLabel = (props: CaptionLabelProps) => {
+  const styles = getStyles();
+  const {
+    fromDate,
+    toDate,
+    formatters: { formatCaption },
+    locale,
+    numberOfMonths,
+  } = useDayPicker();
+
+  const isVisuallyHidden = Boolean(fromDate && toDate && numberOfMonths === 1);
+
+  return (
+    <Heading
+      as="h2"
+      aria-live="polite"
+      aria-atomic="true"
+      id={props.id}
+      className={cx(styles.caption_label, isVisuallyHidden && styles.vhidden)}
+    >
+      {formatCaption(props.displayMonth, { locale })}
```

Comment:
Should we include the year there as well?

<img width="1147" alt="Screenshot 2022-06-02 at 17 14 02" src="https://user-images.githubusercontent.com/6163988/171662719-42f4da00-7515-40db-a76d-11651ca611a0.png">

Reply:
I don't think it's necessary. When you focus calendar with VoiceOver it tells you what month and year it is, then it tells you the chosen date. The [W3C](https://w3c.github.io/aria-practices/examples/dialog-modal/datepicker-dialog.html) example just reads numbers when you navigate between dates.

TLDR: if it was easy to change I would do it, but we will have to customize the whole Day component. So it doesn't really const the efforts, IMO